nvflare/private/fed/server/fed_server.py

```diff
@@ -374,10 +376,23 @@ def _listen_command(self, request: Message) -> Message:
                 reply = make_cellnet_reply(F3ReturnCode.OK, "", None)
                 return reply
         elif command == ServerCommandNames.HEARTBEAT:
+            if job_id not in self.engine.run_processes:
```

Comment:
No, this means the job is not in the parent's job running list, however, the job is still sending heartbeat calls to the parent. In this case, we need to abort the child job process. The network interrupt does not affect this.

For example, the job supposed to be aborted at the server, or the admin action. However, the job child process didn't get the abort command in time, or didn't get it due to network interruption, this will ensure to send the abort command to the child process again.